services/brig/src/Brig/API/Internal.hs

```diff
@@ -278,7 +285,9 @@ authAPI =
   Named @"legalhold-login" (callsFed (exposeAnnotations legalHoldLogin))
     :<|> Named @"sso-login" (callsFed (exposeAnnotations ssoLogin))
     :<|> Named @"login-code" getLoginCode
-    :<|> Named @"reauthenticate" reauthenticate
+    -- We qualify in place to avoid changing the internal API too much
+    -- FUTUREWORK?
```

Comment:
```suggestion
    -- Qualify in place, because the domain is provided in a non-standard way on this end-point, and we don't want to mess with that interface.
```